src/redux/middleware/apiMiddleware.js

```diff
@@ -40,6 +41,11 @@ export const apiCall = (
   meta: { endpoint, ...meta },
 });
 
+const lastLocation = {
```

Comment:
What is this for?

Reply:
Well, there is a mechanism that aborts all pending fetch requests, if the location is changed. Unfortunately, multiple LOCATION_CHANGED actions may be dispatched with the same URL (so not all of them change the location). We need to remember last location, so we can determine, whether the location is actually being changed when the action is dispatched.

Reply:
ah, thanks!